ts/components/ui/Markdown/LoadingSkeleton.tsx

```diff
@@ -19,77 +35,19 @@ export const LoadingSkeleton = ({ testID }: LoadingSkeletonProps) => (
       importantForAccessibility="no-hide-descendants"
       accessibilityElementsHidden={true}
     >
-      <Placeholder.Box
-        width={"98%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"86%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"92%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"80%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"90%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"96%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"84%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"88%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
-      <VSpacer size={8} />
-      <Placeholder.Box
-        width={"94%"}
-        animate={"fade"}
-        color={IOColors["grey-100"]}
-        height={21}
-        radius={4}
-      />
+      {[...Array(lines)].map((_, i) => (
```

Comment:
what if we need a 10+ lines skeleton? 
I believe that leaving the line count to a non-regulated prop with the current implementation leaves the door open for a bunch of bugs.
what do you think?

Reply:
Nice catch, I forgot the reminder operator on the array index

Reply:
that is actually a very witty implementation, I never thought about that so props to you!